jmx-metrics/docs/target-systems/kafka.md

```diff
@@ -7,116 +7,105 @@ These metrics are sourced from Kafka's exposed Yammer metrics for each instance:
 
 ### Broker Metrics
 
-* Name: `kafka.messages.in`
-* Description: Number of messages in per second
-* Unit: `1`
-* Instrument Type: LongValueObserver
+* Name: `kafka.message.count`
+* Description: The number of messages received by the broker
+* Unit: `{messages}`
+* Attributes: `topic`
```

Comment:
I don't believe there are any `topic` attributes after my requested changes (a few instances below as well). Otherwise this lgtm.

Reply:
Good catch, resolved.